src/scenes/PlantsDashboardRouter/components/HighestAndLowestMortalityRateSpeciesCard.tsx

```diff
@@ -35,32 +32,19 @@ export default function HighestAndLowestMortalityRateSpeciesCard(): JSX.Element
 
   useEffect(() => {
     let _highestMortalityRate = 0;
-    let _lowestMortalityRate = isSurvivalRateCalculationEnabled ? Infinity : 100;
+    let _lowestMortalityRate = Infinity;
```

Comment:
We could now change this and line before to be _highestSurvivalRate and _lowestSurvivalRate

Reply:
Yes I'm planning on a later PR to rename everything, because even the file name mentioned mortality rate